Add argument to inhibit using sysctl

Tim-Anderson opened the pull request:
Containers cannot handle running sysctl since this
is a host wide operation.

To correctly operate the VPN you will need to:
 $ sudo sysctl -q -w net.ipv4.ip_forward=1

On the container host before starting and do:
 $ sudo sysctl -q -w net.ipv4.ip_forward=0

After stopping the container.

Signed-off-by: Tim Anderson <tim.anderson@foundries.io>

Tim-Anderson committed:
Add argument to inhibit using sysctl

Containers cannot handle running sysctl since this
is a host wide operation.

To correctly operate the VPN you will need to:
 $ sudo sysctl -q -w net.ipv4.ip_forward=1

On the container host before starting and do:
 $ sudo sysctl -q -w net.ipv4.ip_forward=0

After stopping the container.

Signed-off-by: Tim Anderson <tim.anderson@foundries.io>

doanac commented on factory-wireguard.py:
```diff
@@ -200,7 +201,9 @@ def gen_conf(self, factory: str) -> str:
-    def apply_conf(self, factory: str, conf: str, intf_name: str):
+    def apply_conf(self, factory: str, conf: str, intf_name: str, no_sysctl: bool):
+        if no_sysctl:
+            conf = re.sub(".*sysctl.*\n?","", conf)
```
how about we pass `no_sysctl` to the `gen_conf` method in a way that we won't have to do regex fixes of this file

Tim-Anderson committed:
Change the way removing sysctl is done

This removes the regular expression and moves the logic
to the gen_conf() pair.

Signed-off-by: Tim Anderson <tim.anderson@foundries.io>
